recipe/meta.yaml

```diff
@@ -1,29 +1,43 @@
+{% set version = "1.0.2" %}
+{% set sha256 = "16930239dac17f478e5f5f1e499b77b1e9f1769a8b0b48ed7e7e93f5ffa5a0ca" %}
+
 package:
   name: datreant
-  version: 1.0
+  version: {{ version }}
 
 source:
-   path: .
+  fn: datreant-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/d/datreant/datreant-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
   skip: True  # [win]
 
 requirements:
-  build:
+  host:
     - python
+    - pip
   run:
-    - python
```

Comment:
There is no python dependency on run?

Reply:
I've just seen that. I don't remember why I removed it.

Reply:
Thanks for this. That was the missing link.